rules/windows/process_creation/sysmon_long_powershell_commandline.yml

```diff
@@ -0,0 +1,26 @@
+title: Too Long PowerShell Commandlines
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects Too long PowerShell command lines
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tags:
+    - attack.execution
+    - attack.t1059.001
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/06
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    Powershell_selection:
+      - CommandLine: 
+        - '*powershell*'
+        - '*pwsh*'
+      - Description: 'Windows Powershell'
+      - Product: 'PowerShell Core 6'
+    Length_selection|re:
+      CommandLine: '(.){1000,}'
```

Comment:
you need to put `|re` after the `CommandLine` instead of `Length_selection`. then it's gonna work correctly.

thank you! (:

Reply:
Thanks! Corrected.